frontend/apps/app/app/api/chat/route.ts

```diff
@@ -142,11 +143,17 @@ export async function POST(request: Request) {
   // Convert schema to text
   const schemaText = convertSchemaToText(schemaData)
 
+  // Determine which agent to use based on the question type
+  const questionType = classifyQuestion(message)
+  const agentName =
+    questionType === 'build'
+      ? 'databaseSchemaBuildAgent'
+      : 'databaseSchemaAskAgent'
```

Comment:
ask:

Do you assume that the message will always be in English, at least for now?

Reply:
Yes. 
To remove the limitation of language, in the future we may allow AI to make decisions for us.💭

Reply:
I understand your approach!
I also realize there are several limitations beyond just the language issue:

- Language constraints
- Cases where a term coincidentally matches something (like a table or column name), possibly leading to misclassification
- Handling of negation could be tricky (e.g., “This is not a design question. Is this table about ooo?”)

I also think using AI for classification is one possible approach.
Would love to hear other opinions as well.


--------

The code looks good to me! LGTM!

Reply:
Note: We discussed this within the team and confirmed that it doesn’t seem to be a blocker at this point!

Reply:
I may modify it later to switch between `Ask` and `Build` only by toggling the UI! Let me think about it a bit.🙏